packages/react-components/src/components/chart/utils/getStyles.ts

```diff
@@ -19,6 +29,7 @@ export const getDefaultStyles = (
     lineStyle: 'solid',
     lineThickness: 2,
     yAxis: undefined,
-    significantDigits: 4,
+    significantDigits: significantDigits ?? 4,
```

Comment:
probably worth using same constant across ?there is already Y_AXIS_INTERPOLATED_VALUE_PRECISION. you can update the name and just have one default precision.

Reply:
Also, just for my reference... why 4 ?

Reply:
also, user's can set the precision they want at a chart level and property level. So this value is dynamic, but 4 is the default if unset.

Reply:
right, lets just create a const for this fallback precision value, so we can just use a constant everywhere